assets/src/apps/authoring/components/Flowchart/toolbar/FlowchartHeaderNav.tsx

```diff
@@ -205,7 +207,7 @@ export const FlowchartHeaderNav: React.FC<HeaderNavProps> = () => {
     ['KeyZ'],
     { ctrlKey: true },
   );
-  useKeyDown(handlePartPasteClick, ['KeyV'], { ctrlKey: true }, [copiedPart]);
+  useKeyDown(handlePartPasteClick, ['KeyV'], { ctrlKey: true }, [copiedPart, currentActivityTree]);
```

Comment:
If you copy a part and try it to paste on another screen using shortcuts, it wasn't working and was pasting the part on the same screen from where you copied it because `currentActivityTree` was outdated